typeshed_client/finder.py

```diff
@@ -366,7 +373,7 @@ def find_typeshed() -> Path:
     return importlib_resources.files("typeshed_client") / "typeshed"
 
 
-def parse_stub_file(path: Path) -> ast.Module:
+def ast_parse_file(path: Path) -> ast.Module:
```

Comment:
Let's keep the name `parse_stub_file` as an alias in case any users of the library are using the old name, since it's public.

Reply:
Certainly, I will change it back.

Reply:
Done.

Reply:
I changed it back. But, did you mean to keep the new name but just do `parse_stub_file = ast_parse_file` for backward compatibility?